src/Broadway/EventStore/Exception/DuplicatePlayheadException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Broadway\EventStore\Exception;
+
+use Broadway\EventStore\EventStoreException;
+use Exception;
+
+class DuplicatePlayheadException extends EventStoreException
+{
+    public static function create(Exception $exception)
```

Comment:
I like the idea, and it would actually be pretty easy to implement. However, if we are considering this, shouldn't we just provide the uncommitted events?